commit 573d772c92d30304362ae559c448981d532d56f0
Merge: 74effed 3f7ae9a
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Wed Apr 6 20:41:14 2016 -0700

    Merge pull request #3416 from paulusjacobus/RCBugFix
    
    Optimised Ducth messages to fit in 16 Char display Update language_nl.h

diff --git a/Marlin/language_nl.h b/Marlin/language_nl.h
index 889afb3..4ccc7f0 100644
--- a/Marlin/language_nl.h
+++ b/Marlin/language_nl.h
@@ -153,21 +153,21 @@
 #define MSG_BED_Z                           "Bed Z"
 #define MSG_A_TRAVEL                        "A-travel"
 #define MSG_HEATING_FAILED_LCD              "voorverw. fout"
-#define MSG_ERR_REDUNDANT_TEMP              "Fout: Redundant temp fout"
-#define MSG_THERMAL_RUNAWAY                 "Thermische wegloop"
-#define MSG_ERR_MAXTEMP                     "Fout: Max. temp"
-#define MSG_ERR_MINTEMP                     "Fout: Min. temp"
-#define MSG_ERR_MAXTEMP_BED                 "Fout: Max. temp bed"
-#define MSG_ERR_MINTEMP_BED                 "Fout: Min. temp bed"
+#define MSG_ERR_REDUNDANT_TEMP              "Redun. temp fout"
+#define MSG_THERMAL_RUNAWAY                 "Therm. wegloop"
+#define MSG_ERR_MAXTEMP                     "Err: Max. temp"
+#define MSG_ERR_MINTEMP                     "Err: Min. temp"
+#define MSG_ERR_MAXTEMP_BED                 "Err: Max.tmp bed"
+#define MSG_ERR_MINTEMP_BED                 "Err: Min.tmp bed"
 #define MSG_HEATING                         "Voorwarmen..."
-#define MSG_HEATING_COMPLETE                "voorverw. kompleet"
-#define MSG_BED_HEATING                     "Bed voorverwarmen"
+#define MSG_HEATING_COMPLETE                "Voorverw. kompl."
+#define MSG_BED_HEATING                     "Bed voorverw."
 #define MSG_BED_DONE                        "Bed is voorverw."
 #define MSG_LEVEL_BED_HOMING                "Homing XYZ"
 #define MSG_LEVEL_BED_WAITING               "Klik voor begin"
-#define MSG_LEVEL_BED_DONE                  "Bed leveling kompleet!"
-#define MSG_LEVEL_BED_CANCEL                "Bed leveling afbreken"
-#define MSG_HOME_OFFSETS_APPLIED            "Home offsets toegepast"
+#define MSG_LEVEL_BED_DONE                  "Bed level kompl."
+#define MSG_LEVEL_BED_CANCEL                "Bed level afbr."
+#define MSG_HOME_OFFSETS_APPLIED            "H offset toegep."
 
 #if ENABLED(DELTA_CALIBRATION_MENU)
   #define MSG_DELTA_CALIBRATE               "Delta Calibratie"
